.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "features": {
+    },
+  "postCreateCommand": "python3 -m venv .venv & source .venv/bin/activate & pip3 install --upgrade pip & pip3 install -r requirements.txt",
+  "postStartCommand": "mkdocs serve"
```

Comment:
One thing I don't super like here is the fact that the log of the `mkdocs serve` is not immediately accessible to the user. It can contain a lot of important informations for debugging the documentation build.

I don't really know how to overcome this issue though. Maybe creating a [Task](https://code.visualstudio.com/docs/editor/tasks) and then running it in the `postStartCommand`?

This is just a minor improvement. For a first iteration we can leave it as is (i.e. `mkdocs serve` in the background)